packages/transaction-controller/src/utils/gas.ts

```diff
@@ -136,8 +132,12 @@ async function getGas(
     request.ethQuery,
   );
 
-  if (isCustomNetwork) {
-    log('Using original estimate as custom network');
+  if (isCustomNetwork || simulationFails) {
```

Comment:
Does this mean we were buffering the fallback also? So effectively using 150% of 95% of the block gas limit...

Reply:
Exactly that, it was not taking into account if the gas estimation failed or not.